### ISSUE TITLE
Facility Locator accessibility review

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#12810]
## Description
A maintenance accessibility audit should be done of the Facility Locator to prep for the upcoming Q2 redesign.

### Additional info
Components that should be included in the audit
  - https://www.va.gov/find-locations
  - Search/filtering (Location search, Facility type dropdown, Service type dropdown)
  - Map
  - Search results
  - Pagination
  - Mobile & Desktop experience
 

## Acceptance Criteria
- [x] Complete audit of Facility Locator product
- [x] Document any accessibility issues that should be resolved, improved upon, or considered during the redesign

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [ ] `Public Websites`
- [x] `Facilities`
- [ ] `User support`
